Source/Blazorise/Components/FilePicker/FilePicker.razor.cs

```diff
@@ -434,6 +440,38 @@ protected Task OnProgressed( FileProgressedEventArgs fileProgressedEventArgs )
         /// </summary>
         [Parameter] public bool DisableProgressReport { get; set; } = false;
 
+        /// <summary>
+        /// Gets or Sets whether file picker should upload directories.
+        /// </summary>
+        [Parameter]
+        public bool Directory
+        {
+            get
+            {
+                return directory;
+            }
+            set
+            {
+                if ( value )
+                {
+                    if ( !Attributes?.ContainsKey( "webkitdirectory" ) ?? false )
```

Comment:
Any reason why we need to manually add `webkitdirectory` to the Attributes? Why not just add it directly to the file input element in HTML/razor?

Reply:
I was trying not to change anything on FileEdit. I wasnt sure of what the use case is for FileEdit compared to FilePicker. I will make the changes to support Directory on FileEdit directly.

Reply:
This was removed